tests/test_cssselect.py

```diff
@@ -505,6 +511,13 @@ def xpath(css):
             "e[not(count(preceding-sibling::*) mod 2 = 0)]")
         assert xpath('e:nOT(*)') == (
             "e[0]")  # never matches
+        assert xpath("e:has(> f)") == "e[./f]"
+        assert xpath("e:has(f)") == "e[descendant::f]"
+        assert xpath("e:has(~ f)") == "e[following-sibling::f]"
+        assert (
+            xpath("e:has(+ f)")
+            == "e[following-sibling::*[(name() = 'f') and (position() = 1)]]"
```

Comment:
Would there be a difference between this and `e[following-sibling::f[position() = 1]]`?

Reply:
Update: please see https://github.com/annbgn/cssselect/pull/1